src/include/catalog/catalog_defs.h

```diff
@@ -5,5 +5,8 @@
 namespace terrier::catalog {
 STRONG_TYPEDEF(col_oid_t, uint32_t);
 STRONG_TYPEDEF(db_oid_t, uint32_t);
+STRONG_TYPEDEF(index_oid_t, uint32_t);
+STRONG_TYPEDEF(indexkeycol_oid_t, uint32_t);
```

Comment:
I have no idea what to call this. Should probably rename it before merging.

Reply:
I too have no idea what to call this..

Reply:
Should we name it `indkey_oid_t` as in pg_index? https://www.postgresql.org/docs/current/catalog-pg-index.html